src/cli.rs

```diff
@@ -394,14 +470,18 @@ impl Cli {
         let matches = app.try_get_matches_from_mut(iter)?;
         let mut cli = Self::from_arg_matches(&matches)?;
 
+        // opt-out of claps auto-generated possible values for --pretty
+        // as we already list them in the long_help
+        app = app.mut_arg("pretty", |a| a.hide_possible_values(true));
```

Comment:
**Note to self**: figure out how to restore the order of options  in manpages after mutating one of them them